ASP_tests.py

```diff
@@ -273,43 +277,70 @@ def test_make_regular_grid():
     test_dec = np.array([-44.26396874, -44.26391831, -44.26386787,
                          -44.26389673, -44.26384629, -44.26379586,
                          -44.26382471, -44.26377428, -44.26372384])
-    assert np.allclose(ra_grid, test_ra, atol=1e-7), "RA vals do not match"
-    assert np.allclose(dec_grid, test_dec, atol=1e-7), "Dec vals do not match"
+    assert np.allclose(ra_grid, test_ra, atol=1e-9), "RA vals do not match for Galsim WCS"
+    assert np.allclose(dec_grid, test_dec, atol=1e-9), "Dec vals do not match for Galsim WCS"
+
+    wcs = WCS(wcs_dict)
+    ra_grid2, dec_grid2 = make_regular_grid(ra_center, dec_center, wcs,
+                                      size=25, spacing=3.0)
+    assert np.allclose(ra_grid2, test_ra, atol=1e-9), "RA vals do not match for Snappl WCS"
```

Comment:
I don't think it really matters here, but consider if doing an RA comparison dividing by cos(dec) would better capture the idea of making RA comparisons.